src/components/App.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+export default class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.props = props;
+  }
+
+  render() {
+    return (
+      <div className="app-panel">
+        <h1>Hello from App</h1>
+        <Display />
+        <ButtonPanel />
+      </div>
```

Comment:
- [ ] [REQUIRED] `App` children should be wrapped in [React fragments](https://reactjs.org/docs/fragments.html#short-syntax)
![image](https://user-images.githubusercontent.com/73254040/100472482-f0a2c880-30a1-11eb-9f7e-04631614abb4.png)
[Source](https://microverse.pathwright.com/auth/sign-in/?next=/library/fast-track-curriculum/69047/path/step/44896092/)